.pullapprove.yml

```diff
@@ -44,7 +44,7 @@ groups:
                 - "HTTP"
             files:
                 - "proposed/http-factory/*"
-                - "proposed/http-middleware/*"
+                - "proposed/http-handlers/*"
```

Comment:
When the directories are changed this breaks the link . Eg : https://groups.google.com/d/msg/php-fig/V12AAcT_SxE/8UTIIpzXAAAJ . 

Either we need to keep a README file in the place stating where it is moved or with necessary information.